docs/openapi/schemas.yaml

```diff
@@ -1475,3 +1475,48 @@ BrokenBacklink:
     url_suggested:
       description: The URL suggested for redirecting from the broken backlink.
       $ref: '#/URL'
+ImportAssistantRequest:
+  type: object
+  required:
+    - url
+    - command
+  properties:
+    url:
+      description: The URL to analyze
+      $ref: '#/URL'
+    command:
+      description: Assistant command to execute.
+      type: string
+    prompt:
+      description: Optional prompt to provide context for the analysis.
+      type: string
+    options:
+      description: Optional configuration options required by the command.
+      type: object
```

Comment:
So `name` will be included in 'options' if required?  Could `prompt` be in 'options' as well, or is it complicated enough to deserve a distinct parameter.